shoop/core/models/addresses.py

```diff
@@ -54,51 +55,10 @@ class Labels:
         ENABLED = _('enabled')
 
 
-class AddressManager(models.Manager):
```

Comment:
AddressManager's removal should occur in a commit of its own, to make it easier to find for the changelog.